### PR TITLE
RUMM-3290 Add Error Tracking page for Roku

### PR DESCRIPTION
**Pull request by xgouchet**
### What does this PR do?

Add the Error Tracking documentation for our Roku RUM integration

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by buraizu** [comment]
Thanks, created DOCS-5687 for documentation team review

File: content/en/real_user_monitoring/error_tracking/roku.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+Your crash reports appear in [**Error Tracking**][1].
+
+## Setup
+
+If you have not set up the Roku SDK yet, follow the [in-app setup instructions][2] or see the [Roku RUM setup documentation][3].
+
+1. Add the latest version of the [RUM Roku SDK][4] to your ROPM dependencies (or download the zip archive).
+2. Configure your application's `env` when [initializing the SDK][5].
+3. Run the Gradle task to upload your Proguard/R8 mapping file to Datadog in order to access deobfuscated stack traces.
```
Comment (timmld) [inline content/en/real_user_monitoring/error_tracking/roku.md:38]: is this still valid for Roku?

**Review by timmld — CHANGES_REQUESTED** [review]

**Comment by timmld** [comment]
@xgouchet just one small thing, i never know if the fact that i'm not seeing all the sections in this side navigation is normal in staging or not? Screenshot 2023-06-20 at 13 25 42

File: content/en/real_user_monitoring/error_tracking/roku.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+If you have not set up the Roku SDK yet, follow the [in-app setup instructions][2] or see the [Roku RUM setup documentation][3].
+
+1. Add the latest version of the [RUM Roku SDK][4] to your ROPM dependencies (or download the zip archive).
+2. Configure your application's `env` when [initializing the SDK][5].
+
+For any given error, you can access the file path, line number, and a code snippet for each frame of the related stack trace.
+
+## Forward errors to Datadog
+
+Whenever you perform an operation that might throw an exception, you can forward the error to Datadog as follows:
```
Comment (rtrieu) [inline content/en/real_user_monitoring/error_tracking/roku.md:43]:
```suggestion
Whenever you perform an operation that might throw an exception, you can forward the error to Datadog by adding the following code snippet:
```

Comment (rtrieu) [reply content/en/real_user_monitoring/error_tracking/roku.md:43]: Not sure if this is a code snippet or command, so please edit as necessary.

**Review by rtrieu — CHANGES_REQUESTED** [review]
A few minor suggestions, but otherwise looks good.

**Review by timmld — APPROVED** [review]

**Review by rtrieu — APPROVED** [review]
Looks good!

**Pull request merged**